modules/plugin/geopkg/src/test/java/org/geotools/geopkg/GeoPackageTest.java

```diff
@@ -790,6 +788,25 @@ URL setUpShapefile() throws Exception {
         return URLs.fileToUrl(new File(d, "bugsites.shp"));
     }
 
+    URL setUpBoolShapefile() throws Exception {
+        File d = File.createTempFile("ogrgeojson_8b0dc300", "shp", new File("target"));
+        d.delete();
+        d.mkdirs();
+
+        String[] exts = new String[]{"shp", "shx", "dbf", "prj"};
+        for (String ext : exts) {
+            if("prj".equals(ext)) {
+                String wkt = CRS.decode("EPSG:4326", true).toWKT();
+                FileUtils.writeStringToFile(new File(d, "ogrgeojson_8b0dc300.prj"), wkt);
```

Comment:
There seems to be a bunch of unneccessary manual work (Decoding the WKT, creating the projection file, etc.) going on here. Can't you just copy all the test data to the temp folder?